src/web/app/pages-admin/admin-notifications-page/admin-notifications-page.component.ts

```diff
@@ -0,0 +1,268 @@
+import { Component, OnInit } from '@angular/core';
+import moment from 'moment-timezone';
+import { finalize } from 'rxjs/operators';
+import { NotificationService } from '../../../services/notification.service';
+import { StatusMessageService } from '../../../services/status-message.service';
+import { TableComparatorService } from '../../../services/table-comparator.service';
+import { TimezoneService } from '../../../services/timezone.service';
+import { Notification, Notifications } from '../../../types/api-output';
+import { NotificationType, NotificationTargetUser } from '../../../types/api-request';
+import { SortBy, SortOrder } from '../../../types/sort-properties';
+import { collapseAnim } from '../../components/teammates-common/collapse-anim';
+import { ErrorMessageOutput } from '../../error-message-output';
+import {
+  NotificationEditFormMode,
+  NotificationEditFormModel,
+} from './notification-edit-form/notification-edit-form-model';
+import { NotificationsTableRowModel } from './notifications-table/notifications-table-model';
+
+// default sorting order for rows
+const defaultNotificationsTableRowModelsSortOrder = SortOrder.ASC;
+
```

Comment:
It makes sense to me. See how prof wants this to be. Note that if that is the case, the default order of the left most column (#) will be in reversed ordering. Like this:

![image](https://user-images.githubusercontent.com/10681776/158587801-5f24a14f-36b3-4a61-ab86-a5ec16a3dd56.png)

Reply:
Yup let's wait for prof @damithc 's view, as to whether it adds much value to his user experience by having the latest created notifications on top by default. My main concern is if the list gets long newer notifications will get pushed back in the list and the user will need to scroll down every time if he were to edit/delete notification. But ofcourse he can also reverse the list by create time manually, which although will require one more extra step, may not be a big deal.

Reply:
Yes, newest on top sounds reasonable. 👍 
Plus, this kind of things can be tweaked post-release, after we actually experience the feature in action for a while.

Reply:
@damithc Thanks for your input.

@fsgmhoward if it is not too much effort, you can tweak the implementation a bit to allow for newest on top. Otherwise, we can take note of this and come back to it again later as a possible UX improvement.

Reply:
Regarding the leftmost # column, I realise if the index contain many digits it might cause a problem like this:
<img width="732" alt="Screenshot 2022-03-16 at 11 14 52 PM" src="https://user-images.githubusercontent.com/59337195/158623868-b828c5d5-0f44-4639-bb2d-8d314b897052.png">

Maybe we can omit the # column since the user can already identify each row with the notification title / start time.

Reply:
Sure. We can omit it. I was there for visual purposes only (like also to provide user a way to sort them in creation order).
Alternatively, I can still preserve this column. This style issue can be fixed I think.

Reply:
> Do you think it is still necessary to let them sort by creation order?

It would be good to be able to do so. I am thinking of replacing the index column with a column for date created itself so that we can avoid dealing with large index numbers. But maybe let's leave the index column as it is first, as we come back again to improve the UX.

Reply:
I did an investigation and this might be due to the screen width being too small. Nevertheless, I changed back to datebrief so the date will not take up too much of the width. The full date is still visible via the tooltip. Demo:
![image](https://user-images.githubusercontent.com/10681776/158759410-67059351-6fe7-4854-ba97-c7fc94880076.png)

It save quite some space for this index number and for the title. However, the title can be long so we will need a PR later to cut it short for preview only as a UI improvement.